### PR TITLE
add more details to README.md

### PR DESCRIPTION
**Pull request by lnnrtwttkhn**
Hi @EstherPlomp,

I realized that the README.md was still quite empty.
This PR adds a bit more information.
Still not as complete as one would maybe like, but a good start.

Feel free to add any missing information. Thanks!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,77 @@
...
+
+## About
+
+Slides for talk on "Reproducible & FAIR Teaching Materials" by Esther Plomp and Lennart Wittkuhn.
+Prepared for the Digital Research Academy train-the-trainer program.
+
+## Requirements
+
+- [Quarto](https://quarto.org/)
+- [renv](https://rstudio.github.io/renv/articles/renv.html)
```
Comment (EstherPlomp) [inline README.md]: Should we add R / RStudio?

File: README.md [hunk 2]
```diff
@@ -1,3 +1,77 @@
...
+
+```
+</details>
+
+## License
+
+Creative Commons Attribution 4.0 International ([CC BY 4.0](https://creativecommons.org/licenses/by/4.0/)).
+For details, see the [`LICENSE.md`](LICENSE.md) file.
+
+## Contributors
```
Comment (EstherPlomp) [inline README.md:75]: Maybe a citation placeholder since we don't have that in place yet?

File: README.md [hunk 3]
```diff
@@ -1,3 +1,77 @@
...
+# Reproducible & FAIR Teaching Materials
 
-[![Quarto Publish](https://github.com/lnnrtwttkhn/dra-fair-teaching/actions/workflows/publish.yml/badge.svg)](https://github.com/lnnrtwttkhn/dra-fair-teaching/actions/workflows/publish.yml)
\ No newline at end of file
+[![Quarto Publish](https://github.com/lnnrtwttkhn/dra-fair-teaching/actions/workflows/publish.yml/badge.svg)](https://github.com/lnnrtwttkhn/dra-fair-teaching/actions/workflows/publish.yml)
+
+## About
+
+Slides for talk on "Reproducible & FAIR Teaching Materials" by Esther Plomp and Lennart Wittkuhn.
+Prepared for the Digital Research Academy train-the-trainer program.
```
Comment (EstherPlomp) [inline README.md]:
```suggestion
Prepared for the [Digital Research Academy](https://digital-research.academy/) [train-the-trainer program](https://gitlab.com/digital-research-academy/dra-train-the-trainer).
```

File: README.md [hunk 4]
```diff
@@ -1,3 +1,77 @@
-# dra-fair-teaching
+# Reproducible & FAIR Teaching Materials
 
-[![Quarto Publish](https://github.com/lnnrtwttkhn/dra-fair-teaching/actions/workflows/publish.yml/badge.svg)](https://github.com/lnnrtwttkhn/dra-fair-teaching/actions/workflows/publish.yml)
\ No newline at end of file
+[![Quarto Publish](https://github.com/lnnrtwttkhn/dra-fair-teaching/actions/workflows/publish.yml/badge.svg)](https://github.com/lnnrtwttkhn/dra-fair-teaching/actions/workflows/publish.yml)
+
+## About
+
+Slides for talk on "Reproducible & FAIR Teaching Materials" by Esther Plomp and Lennart Wittkuhn.
```
Comment (EstherPlomp) [inline README.md]:
```suggestion
Slides for talk on "Reproducible & FAIR Teaching Materials" by Esther Plomp and Lennart Wittkuhn on 29 August 2023.
```

**Review by EstherPlomp — APPROVED** [review]
Thanks for adding - it was indeed a bit empty. 

I did some nitpicking: I don't think anything major is missing.

File: README.md (hunk 2 above)
Comment (lnnrtwttkhn) [reply README.md:75]: added this to a new issue, see #14

File: README.md (hunk 1 above)
Comment (lnnrtwttkhn) [reply README.md]: good idea! added in https://github.com/lnnrtwttkhn/dra-fair-teaching/pull/13/commits/255182b475ef720a50eeeafa8cf3fd2d0b721ab7

**Comment by lnnrtwttkhn** [comment]
Thanks for all the comments @EstherPlomp. Will merge this now.

**Pull request merged**